content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst

```diff
@@ -1,7 +1,19 @@
+:show-content:
+:hide-page-toc:
+
 ==================
 Removal strategies
 ==================
 
+.. toctree::
+   :titlesonly:
+
+   removal_strategies/fifo
+   removal_strategies/lifo
+   removal_strategies/fefo
+   removal_strategies/closest_location
+   removal_strategies/least_packages
+
```

Comment:
minor suggestion: put the toctree at the end of the document, as it is usually what we do on pages that have both content to display and children pages.